app/views/content/is-teaching-right-for-me/maths/maths/_article.html.erb

```diff
@@ -0,0 +1,95 @@
+      <ul class="vertical-tags">
+        <li class="tag">Numbers</li>
+        <li class="tag">Algebra</li>
+        <li class="tag">Ratio, proportion and rates of change</li>
+        <li class="tag">Geometry and measures</li>
+        <li class="tag">Probability and statistics</li>
+      </ul>
+
+      <p>Themes you’ll cover for 16 to 18 year olds (key stage 5) - A or AS level core maths:</p>
+      <ul class="vertical-tags">
+        <li class="tag">Analysis of data</li>
+        <li class="tag">Maths for personal finance</li>
+        <li class="tag">Estimation</li>
+        <li class="tag">Critical analysis of given data and models</li>
+      </ul>
+    <p><a href="https://www.gov.uk/government/publications/national-curriculum-in-england-mathematics-programmes-of-study/national-curriculum-in-england-mathematics-programmes-of-study#key-stage-3">Read the full national curriculum for more information about teaching maths in secondary schools</a>.</p>
+    <p>You may also get the opportunity to teach other subjects, particularly for 11 to 16 year olds and in some cases for post-16. This will vary depending on individual schools.
+    </div>
+</section>
+<section class="col col-845 inset">
+    <%= render Content::PhotoQuoteListComponent.new(
+      [
+        {
+          image: "static/images/content/subjects/rotna-roy.jpeg",
+          heading: "Empowering my pupils",
+          text: "I enjoy showing my pupils that maths is essential in everyday life. Whether it is identifying patterns, interpreting data, managing finances, or understanding the spaces and structures around us, everything uses maths.",
+          accreditation: "Rotna Roy, Maths Lead at a school in London",
+        }
+      ],
+      colors: %w[yellow],
+      numbered: false
+    ) %>
+  </section>
+
+<section class="col col-space-m col-720">
+<h2>Fund your teacher training</h2>
+<p>To train to be a maths teacher, you will need to do a teacher training course. </p>
+<p>Tax-free bursaries of £28,000 and scholarships of £30,000 are available for <a href="/funding-and-support/scholarships-and-bursaries?funding_widget%5Bsubject%5D=maths&button=#funding-widget">eligible trainee maths teachers</a>. </p>
+<p>You can find out more about <a href="https://teachingmathsscholars.org/home">the maths scholarship from The Institute of Mathematics and its Applications, and partners</a>.</p>
+<p>You can receive this alongside a tuition fee and maintenance loan. </p>
+<p>You may also be able to get extra funding support if you are: </p>
+<ul>
+<li>a parent or carer </li>
+<li>disabled </li>
+</ul>
+<p>Non-UK citizens without indefinite leave to remain in the UK are unlikely to be eligible for funding.</p>
+<h2>Check your qualifications</h2>
+<p>You'll need a bachelors degree before training to teach, but it does not have to be in maths.</p>
+<p>Besides maths, the most popular degrees people have before doing a teacher training course in maths are:</p>
```

Comment:
@gemmadallmandfe @caitlin-reidy I think it would be good to show the variation as well as the most popular. I agree with your earlier comment Caitlin that we don't want to give unrealistic expectations. But at the same time there are some subjects that may have had a number of successful candidates that did degrees people may not automatically associate with training to be a maths teacher.

And I also see Gemma's point that if we list them as bullet points it may suggest, as people tend to skim, that these are the main/ only subjects you need a degree in to train to be a maths teacher. So perhaps it may be better to have it a a sentence?

Reply:
@MaxineCl @gemmadallmandfe Moving away from bulletpoints isn't very accessible but happy to do it if you guys want. In terms of including variety, do you want random courses (that appear v different to maths ie. theatre) picked for the list regardless of how small number of users from them?